sabnzbd/downloader.py

```diff
@@ -372,12 +372,17 @@ def remove_socket(self, nw: NewsWrapper):
     @NzbQueueLocker
     def set_paused_state(self, state: bool):
         """Set downloader to specified paused state"""
-        self.paused = state
+        if self.paused != state:
+            if cfg.preserve_state():
+                cfg.start_paused.set(int(state))
+            self.paused = state
 
     @NzbQueueLocker
     def resume(self):
         # Do not notify when SABnzbd is still starting
         if self.paused and sabnzbd.WEB_DIR:
+            if cfg.preserve_state():
+                cfg.start_paused.set(0)
```

Comment:
`start_paused` is an `OptionBool` so use `.set(False)` and `.set(True)`.
And let's put it below the `sabnzbd.notifier.send_notification`